### PR TITLE
Rerender feedstock; bump build

### PR DESCRIPTION
**Pull request by alimanfoo**
...to get PY36 build.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -31,7 +34,7 @@ test:
 
 about:
   home: https://github.com/harlowja/fasteners
-  license: Apache License Version 2.0
+  license: Apache Version 2.0
```
Comment (jakirkham) [inline recipe/meta.yaml]: Also could drop "Version".

While we are at it, could we please add `license_file: LICENSE` to package the license from the source code?

Comment (alimanfoo) [reply recipe/meta.yaml]: Sure.

**Comment by alimanfoo** [comment]
@jakirkham looks like I will need to rerender feedstock again to get PY36. I just tried this locally and it added PY36 for circle and appveyor but not for travis. I'm guessing it's because there is a dependency on monotonic which I only updated yesterday, and travis is slow at the moment, so maybe I just need to wait?

**Comment by jakirkham** [comment]
Yeah waiting would be the typical way to solve it. Travis has been pretty slow during the week, but picks up during the weekend.

Alternatively you could hack it in. Should be a copy paste with an obvious tweak.

**Pull request merged**